tests/test_utils.py

```diff
@@ -51,3 +51,16 @@ def test_convert_datetime_to_ui_str_format(self):
         current_time = datetime(year=2021, month=8, day=30, hour=15, minute=30, second=45)
         current_time_str = convert_datetime_to_ui_str_format(current_time)
         assert current_time_str == "August 30 2021, 15:30 h"
+
+    def test_str_to_pendulum_datetime(self):
+        datetime_obj = datetime(year=2021, month=4, day=5, hour=12, minute=30)
+        assert str_to_pendulum_datetime("2021-04-05T12:30") == datetime_obj
+        assert str_to_pendulum_datetime("2021-04-05T12:30:00") == datetime_obj
+        datetime_obj = today(tz="UTC")
+        datetime_obj = datetime_obj.set(hour=12, minute=30)
+        assert str_to_pendulum_datetime("12:30") == datetime_obj
+        assert str_to_pendulum_datetime("12:30:00") == datetime_obj
+        with pytest.raises(Exception):
```

Comment:
Nit, it's better if we explicitly check the `ValueError` exception instead of a broad Exception

Reply:
The code (both original and updated) raise an Exception, not a ValueError. ValueError is the exception that is emitted by from_format and caught in the method

Reply:
Ah you're right, LGTM then